### ISSUE TITLE
count(): Parameter must be an array 

### ISSUE THREAD

**Issue by netwrkx** [viralsolani/laravel-adminpanel#96]
hello, for unknown reason dd($assets); return null

ErrorException (E_ERROR)
count(): Parameter must be an array or an object that implements Countable (View: C:\laravel\laravel-adminpanel\resources\views\backend\history\partials\item.blade.php) (View: C:\laravel\laravel-adminpanel\resources\views\backend\history\partials\item.blade.php) (View: C:\laravel\laravel-adminpanel\resources\views\backend\history\partials\item.blade.php)

```
public function renderDescription($text, $assets = false)
    {
        $assets = json_decode($assets, true);
        
        $count = 1;
        $asset_count = count($assets) + 1;
```

**Comment by viralsolani** [comment]
Hello @netwrkx, Can you please attach the error screenshot?

**Comment by netwrkx** [comment]
![error](https://user-images.githubusercontent.com/5035044/34326773-797e1b2e-e8b5-11e7-9624-67bad34a2fe4.PNG)

**Comment by bvipul** [comment]
Solved in #101
